packages/core/src/features/facets/facetSlice.ts

```diff
@@ -72,6 +72,8 @@ const slice = createSlice({
                   };
                 } else {
                   // Unhandled aggregation
+                  state.cases[field].status = "fulfilled";
+                  state.cases[field].error = "Unhandled aggregation";
```

Comment:
Given @wteouchicago comment on the message content,  could this be a const string? So we only need to change the constant instead of several?

Reply:
i think its best to keep the error message as "Unhanded aggregation" here but to change it in the user facing component. Can make that a constant

Reply:
made the changes on browser-based editor, so i could not test. Please make sure this works